UnitTests/GitUITests/Editor/FileViewerTextTests.cs

```diff
@@ -41,7 +41,9 @@ public void DoAutoCRLF_should_not_unnecessarily_duplicate_line_ending(AutoCRLFTy
         {
             using (var testHelper = new GitModuleTestHelper())
             {
-                testHelper.CreateRepoFile(@".git\config", $"[core]\n\tautocrlf = {autoCRLFType}");
+                testHelper.Module.LocalConfigFile.SetEnum("core.autocrlf", autoCRLFType);
+                testHelper.Module.LocalConfigFile.Save();
```

Comment:
📝 If this continues to be flaky, the likely cause in the file system not assigning a new timestamp to this save operation, immediately following the first save operation in the constructor.